app/uk/gov/hmrc/customs/declarations/stub/connector/NotificationConnector.scala

```diff
@@ -91,6 +91,7 @@ class NotificationConnector @Inject()(
     val delay = extractDelay(lrn)
     lrn.headOption match {
       case Some('B') => (delay, generator.generate(lrn, Seq(Rejected)).toString)
+      case Some('C') => (delay, generator.generate(lrn, Seq(Cleared)).toString)
```

Comment:
Think you probably should send an `Accepted` and then a `Cleared` to more closely align with what DMS actually send us (maybe double check with Liz)